.github/workflows/greetings.yml

```
@@ -10,4 +10,4 @@ jobs:
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           issue-message: 'Hi there! Thanks for opening your first issue!'
-          pr-message: 'Hi there! Thanks for opening your first pull request!'
+          pr-message: 'Hi there! Thanks for opening your first pull request! Please make sure it references the issue where your proposal was discussed and accepted, as required by CONTRIBUTING.md.'
```

actionlint (1.7.12)

[error] input "pr-message" is not defined in action "actions/first-interaction@v3". available inputs are "issue_message", "pr_message", "repo_token"

(action)